test/parallel/test-util-inspect.js

```diff
@@ -1151,4 +1151,21 @@ if (typeof Symbol !== 'undefined') {
 }
 
 assert.doesNotThrow(() => util.inspect(process));
+
+// @@toStringTag
+assert.strictEqual(util.inspect({ [Symbol.toStringTag]: 'a' }),
+                   'Object [a] { [Symbol(Symbol.toStringTag)]: \'a\' }');
+
+class Foo {
+  constructor() {
+    this.foo = 'bar';
+  }
+
+  get [Symbol.toStringTag]() {
+    return this.foo;
+  }
+}
+
+assert.strictEqual(util.inspect(new Foo()), 'Foo [bar] { foo: \'bar\' }');
+
```

Comment:
Also needed is tests for when `Symbol.toStringTag` is not a string, or when its getter function throws.